core/trino-main/src/main/java/io/trino/operator/scalar/ParametricScalar.java

```diff
@@ -47,7 +47,8 @@ public ParametricScalar(
             Signature signature,
             ScalarHeader details,
             ParametricImplementationsGroup<ParametricScalarImplementation> implementations,
-            boolean deprecated)
+            boolean deprecated,
+            String actualName)
```

Comment:
Let's not pass the actual name but take the name from the signature for now.

Reply:
Do we need to keep argument `actualName` in ctor or just remove it ?

Reply:
We can remove it for now.